meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```
@@ -48,6 +49,9 @@ python do_prepare_mender_shell_conf() {
     if not "ServerURL" in mender_shell_conf:
         mender_shell_conf["ServerURL"] = d.getVar("MENDER_SERVER_URL")
 
+    if not "User" in mender_shell_conf:
```

Suggestion:
I like
```suggestion
    if "User" not in mender_shell_conf:
```
simply because it's like straight up english, but not necessary ofc

---

Nice, thanks. I'll fix, but not re-run tests :+1: